config/routes.rb

```diff
@@ -1,4 +1,15 @@
 Rails.application.routes.draw do
   mount_devise_token_auth_for 'User', at: 'auth'
+  
   root 'static_pages#index'
+  
+  get '/users/:id?includes=orders,services', to: 'users#show'
```

Comment:
get parameters should not be in the route: http://guides.rubyonrails.org/routing.html#the-query-string, the controller will pick up all the params automatically